demo/site/src/documents/pages/Page.tsx

```diff
@@ -149,7 +149,7 @@ export async function Page({ pageTreeNodeId, scope }: { pageTreeNodeId: string;
     return (
         <>
             {document.seo.structuredData && document.seo.structuredData.length > 0 && (
-                <script type="application/ld+json">{document.seo.structuredData}</script>
+                <script type="application/ld+json" dangerouslySetInnerHTML={{ __html: document.seo.structuredData }} />
```

Comment:
I am not 100% sure what's happening here, but we are trying to set some seo relevant data. If possible, I would recommend that we avoid `dangerouslySetInnerHtml` when possible.

Why are we not using [Dynamic Metadata](https://nextjs.org/docs/app/building-your-application/optimizing/metadata#dynamic-metadata)  like mentioned in Next Documentation?

Reply:
ok probably we can not use this, because one can only do this in pure server components.